.github/ISSUE_TEMPLATE/config.yaml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord Community
+    url: https://codesupport.dev/discord
+    about: For general CRT discussion and support please visit our Discord
```

This needs changing:
```suggestion
    about: For general website discussion and support please visit our Discord
```